passes/pmgen/README.md

```diff
@@ -232,5 +235,105 @@ But in some cases it is more natural to utilize the implicit branch statement:
         portAB = \B;
     endcode
 
-There is an implicit `code..endcode` block at the end of each `.pmg` file
-that just accepts everything that gets all the way there.
+There is an implicit `code..endcode` block at the end of each (sub)pattern
+that just rejects.
```

Comment:
What's the reason for this backwards-compatibility-breakage? Is it really necessary?
I don't expect many pmgen-s to be out in the field right now, but we should always be careful when we break backwards compatibility -- particularly since this will break with yosys-0.9.

Reply:
Yes, it's necessary. The only sensible other options would be to create a complex set of rules for when we automatically accept at the end of the pattern, or force people to put explicit `reject` calls all over their code. Neither seems like a good idea to me.